app/src/main/java/com/jerboa/db/AppDBMigrations.kt

```diff
@@ -354,6 +354,123 @@ val MIGRATION_26_25 = object : Migration(26, 25) {
     }
 }
 
+val MIGRATION_26_27 = object : Migration(26, 27) {
```

Comment:
I am not home and doing this on my phone so i can't properly vet this SQL migration. But does this pass the migration test?